packages/vue/src/components/molecules/SfBanner/SfBanner.vue

```diff
@@ -19,17 +19,19 @@
       <slot name="call-to-action" v-bind="{ buttonText }">
         <SfButton
```

Comment:
we still have our previous problem with btn in btn on mobile :) what do you think about adding an additional condition for showing this button only on desktop? :)

Reply:
You are right, I add this condition.